src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -151,6 +151,11 @@ - (nullable MTRDeviceController *)initWithParameters:(MTRDeviceControllerAbstrac
     return nil;
 }
 
+- (void)dealloc
+{
+    MTR_LOG("%@ dealloc: %p", NSStringFromClass([self class]), self);
```

Comment:
Mabye log `self.UUID` too?  Or more generally, log `self` as `%@`, not the class string?

Reply:
Given the current description strings aren't complicated I've changed it to log `self`.